app/controllers/groups_controller.rb

```diff
@@ -13,6 +13,20 @@ def index
 
   def show
     authorize! @group, to: :read?
+
+    respond_to do |format|
+      if params.key? :parent_id
+        format.turbo_stream do
+          @group = Group.find(params[:parent_id])
+          @collapsed = params[:collapse] == 'true'
+          @children = subgroups_and_projects(@group.id)
```

Comment:
**nitpick (blocking):** Can you change this to only call `subgroups_and_projects` when we it is not being collapsed. If collapse is true can you set `@children = Namespace.none`

Reply:
Updated in bba4328